src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelReferenceElementsVisitor.cs

```diff
@@ -15,11 +15,11 @@ namespace Microsoft.OData.Edm.Csdl.Serialization
     /// </summary>
     internal class EdmModelReferenceElementsVisitor
     {
-        private readonly EdmModelCsdlSchemaWriter schemaWriter;
+        private readonly EdmModelCsdlSchemaXmlWriter schemaWriter;
 
         internal EdmModelReferenceElementsVisitor(IEdmModel model, XmlWriter xmlWriter, Version edmxVersion)

```

Comment:
>EdmModelReferenceElementsVisitor [](start = 17, length = 32)

Should this have "Xml" in the name, since it is specific to Xml?  Should we have separate sub-directories for "Xml" and "Json"?

Reply:
I changed as "EdmModelReferenceElementsXmlVisitor".

For the sub-folder, so far, there's not too much cs files. So, i'd like to leave them in the same folder.

---
In reply to: [429420185](https://github.com/OData/odata.net/pull/1775#discussion_r429420185) [](ancestors = 429420185)